### PR TITLE
Erp menu

### PR DESCRIPTION
**Pull request by Daelso**
## About

This is a in-game mechanical erp menu built into the existing strip item menu. It has both consensual and non-con options with harm intent toggled. There is sure to be some bug I missed so woo. Most actions have sounds to go along with them. Code is well-commented (by ss13 standards) so others should be able to mess with it too. Also includes various sound, orgasm and moaning procs.

## Why It's Good For The Game

The birth of many horrors to cum. It is a human-to-human erp menu. Will work on expanding it to aliens in the future.

## Changelog
:cl:
## Adds forbidden fruits menu to ss13 strip menu.
image
## Fondle Menu
image
## Fingering Menu
image
## Oral Menu
image
## Sex Menu
image

## *moan
Adds sounds to the above emote
/:cl:

Ignore the first file, second one is correct.

### CODE REVIEW

File: code/modules/mob/living/carbon/human/human.dm [hunk 1]
```diff
@@ -196,6 +196,8 @@
 	<BR><A href='?src=[REF(src)];pockets=1'>Empty Pockets</A>
 	<BR>
 	<BR><A href='?src=[REF(user)];refresh=1'>Refresh</A>
+	<BR>
+	<BR><A href='?src=[REF(src)];forbidden=1'>Forbidden Fruits</A>
```
Comment (ProbablySpaghetti) [inline code/modules/mob/living/carbon/human/human.dm:200]: Not 100% sure about the name, but eh. This works unless you can think of something better.

File: code/modules/mob/living/carbon/human/human.dm [hunk 2]
```diff
@@ -320,7 +322,15 @@
...
 		if(interactee&&(in_range(src, usr)))
 			show_inv(interactee)
-
+	if (href_list["forbidden"])
+		if(interactee&&(in_range(src, usr)))
+			erp_menu(interactee)
+	if (href_list["slapass"])
+		if(interactee&&(in_range(src, usr)))
+			erp_menu(interactee)
+			visible_message("[usr] slaps [src]'s ass!")
```
Comment (ProbablySpaghetti) [inline code/modules/mob/living/carbon/human/human.dm:331]: Add a span class to the message maybe?

File: code/modules/mob/mob.dm [hunk 3]
```diff
@@ -390,6 +390,14 @@
 	popup.set_content(dat)
 	popup.open()
 
+/mob/proc/erp_menu(mob/user)
+	user.set_interaction(src)
+	var/dat = {"
+	<BR> Test!
+	<BR>"}
```
Comment (ProbablySpaghetti) [inline code/modules/mob/mob.dm:397]: Is this a placeholder, or just a 'something went wrong' message?

Comment (Daelso) [reply code/modules/mob/mob.dm:397]: The proc is needed. As far as I can tell it doesn't do anything and is overwritten by the topic. However, it won't compile without the proc.

File: code/modules/mob/living/carbon/human/human.dm (hunk 1 above)
Comment (Daelso) [reply code/modules/mob/living/carbon/human/human.dm:200]: I just stole it from lifeweb cuz thats what they call their erp menu. It can be called whatever

**Pull request merged**